MSProject/ViewControllers/LibraryTableView/WBLibraryTableViewController.swift

```diff
@@ -125,7 +125,7 @@ class WBLibraryTableViewController: UIViewController {
     
     // MARK: - Services
     @objc private func loadBooks() {
-        guard !WBBooksManager.sharedIntance.needsReload.value && WBBooksManager.sharedIntance.bookViewModels.value.isEmpty else {
+        guard WBBooksManager.sharedIntance.needsReload.value || WBBooksManager.sharedIntance.bookViewModels.value.isEmpty else {
```

Comment:
it was there cause the first time the array its empty, but initializing the needsReload on true and only changing to false when the server response success, it will work without the validation of empty array... 🥳 🤟🏻